openhtf/util/measurements.py

```diff
@@ -336,14 +340,15 @@ def value(self):
     """The values stored in this record.
 
     Returns:
-      A list of tuples; the last element of each tuple will be the measured
-    value, the other elements will be the assocated coordinates.  The tuples
-    are output in the order in which they were set.
+      A list of dicts mapping coordinates to values measured at those
+      coordinates. Elements are output in the order in which they were set.
     """
     if not self.is_value_set:
       raise MeasurementNotSetError('Measurement not yet set', self.name)
-    return [dimensions + (value,) for dimensions, value in
-            self.value_dict.iteritems()]
+    return [{
```

Comment:
The more I see the old way the less I like it. But we can keep it the old way for now because I'm out of time to argue about it, and because you might be right.